ui/src/views/network/VpcTab.vue

```diff
@@ -684,10 +684,19 @@ export default {
 
       switch (e) {
         case 'privateGateways':
-          this.rules = {
-            ipaddress: [{ required: true, message: this.$t('label.required') }],
-            gateway: [{ required: true, message: this.$t('label.required') }],
-            netmask: [{ required: true, message: this.$t('label.required') }]
+          if (this.isAdmin()) {
+            this.rules = {
+              ipaddress: [{ required: true, message: this.$t('label.required') }],
+              gateway: [{ required: true, message: this.$t('label.required') }],
+              netmask: [{ required: true, message: this.$t('label.required') }]
+            }
+          } else {
+            this.rules = {
+              ipaddress: [{ required: true, message: this.$t('label.required') }],
+              gateway: [{ required: true, message: this.$t('label.required') }],
+              netmask: [{ required: true, message: this.$t('label.required') }],
+              associatednetworkid: [{ required: true, message: this.$t('label.required') }]
+            }
           }
```

Comment:
```suggestion
          this.rules = {
            ipaddress: [{ required: true, message: this.$t('label.required') }],
            gateway: [{ required: true, message: this.$t('label.required') }],
            netmask: [{ required: true, message: this.$t('label.required') }]
          }
          if (!this.isAdmin()) {
            this.rules.associatednetworkid = [{ required: true, message: this.$t('label.required') }]
          }
```
@weizhouapache can this be simplified?